src/backup.rs

```diff
@@ -32,6 +51,27 @@ impl MegolmV1BackupKey {
     pub fn backup_algorithm(&self) -> String {
         self.inner.backup_algorithm().into()
     }
+
+    /// Try to create a [`MegolmV1BackupKey`] from a base 64 encoded string.
+    #[napi(strict)]
+    pub fn from_base64(key: String) -> napi::Result<MegolmV1BackupKey> {
+        Ok(Self { inner: InnerMegolmV1BackupKey::from_base64(&key).map_err(into_err)? })
+    }
+
+    #[napi(strict)]
+    pub async fn encrypt(&self, exported_session_json: String) -> napi::Result<KeyBackupData> {
```

Comment:
How does this json get retrieved?

Reply:
Ah, comes from export_room_keys_for_session. I should document that really!